### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-282.2] Text: UIs are truncated at a width equivalent to 320 CSS pixels 

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11296]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
component having issue: https://sap.github.io/fundamental-ngx/#/core/wizard
On accessing wizard in width equivalent to 320 CSS pixels, the labels and the place holder text in the wizard dialog are truncated and not visible completely.
#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
open any wizard with width equivalent to 320 CSS pixels, the labels and the place holder text in the wizard dialog should be truncated and not visible completely.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes
#### Is there anything else we should know?
Attaching screenshot for reference
[ACC-260.2-1.pdf](https://github.com/SAP/fundamental-ngx/files/14002343/ACC-260.2-1.pdf)
[ACC-260.2-2.pdf](https://github.com/SAP/fundamental-ngx/files/14002344/ACC-260.2-2.pdf)

**Comment by nikolay-kolarov** [comment]
Hello @JyotiKumari1507,

I think this duplicated with #11243? Reduced space or zoom both result in less space and truncation, so I think we can close this one and track the other.

Thanks,
Nikolay

**Comment by JyotiKumari1507** [comment]
@droshev both look similar. Difference is the pixel and percentage set of screen during testing. Please go through ticket details again

**Comment by nikolay-kolarov** [comment]
Hello @JyotiKumari1507,

For me, those are just two different ways to reach the same technical limitation, so I do not think the team needs a second item for this. Maybe you can add a comment in https://github.com/SAP/fundamental-ngx/issues/11243 explaining that you can achieve it also by the testing method described here.

Best Regards,
Nikolay
